python/foglamp/services/core/server.py

```diff
@@ -138,7 +138,7 @@ class Server:
         'enableHttp': {
             'description': 'Enable or disable the connection via HTTP',
             'type': 'boolean',
-            'default': 'false'
+            'default': 'true'
```

Comment:
Yes, that is the real change required to make Admin REST API to start with http as default! 👍 

Agree! To avoid confusion, this key should be renamed to enableHttp**s**